pyodide_build/recipe/builder.py

```diff
@@ -596,7 +596,14 @@ def _package_wheel(
             )
             if self.build_metadata.cross_build_env:
                 subprocess.run(
-                    ["pip", "install", "-t", str(host_site_packages), f"{name}=={ver}"],
+                    [
+                        "pip",
+                        "install",
+                        "--upgrade",
```

Comment:
Just curious why we need an `--upgrade` here. I think passing the version explicitly would reinstall the package anyway if there is different version installed.

Reply:
One would think. I think this is a pip bug with the `-t` option, it actually adds the `numpy-2.0.2-dist-info` directory but leaves the `numpy-1.3.2-dist-info` directory too and doesn't update the package.

Reply:
I think it is the intended behaviour. Here is what the `-t` option from `pip install --help` says:

```
Install packages into <dir>. By default this will not replace existing files/folders in
<dir>. Use --upgrade to replace existing packages in <dir> with new versions.
```